app/models/labware_creators/merged_plate.rb

```diff
@@ -76,32 +76,70 @@ def all_source_barcodes_entered?
       return if minimal_barcodes.size == expected_source_purposes.size
 
       msg = 'Please scan in all the required source plate barcodes.'
-      errors.add(:parent, msg)
+      errors.add(:base, msg)
     end
 
-    # Validation to check all source plates have the same parent
-    def source_plates_have_same_parent?
-      return if source_plates.map { |sp| sp.parents.map(&:id) }.flatten.uniq.one?
+    # Validation to check all source plates can be merged.
+    # Currently we allow this is there is a maximum of one request associated with each well
+    # Theoretically we can allow merging of plates with two distinct requests, as long as they have two
+    # different tag sets. See #merge_index for details
+    def source_plates_can_be_merged?
+      expected_merges = expected_merges(source_plates)
+
+      return unless expected_merges.values.any? { |v| v.uniq.many? }
```

Comment:
Not sure what the `v.uniq.many?` is doing here?
So `v` would be, in your example below,  `['1', false]`
So `v.uniq.many?` would be true

Maybe can chat about it on Monday

Reply:
Went over this in the zoom, but, to keep the answer on the record:

This was due to a mistake in the comment about expected_merges, which returns an array of arrays, not just an array. Fix coming with updated comment.